### ISSUE TITLE
No thumbnail / Pas de miniatures

### ISSUE THREAD

**Issue by Blankovitch** [OpenShot/openshot-qt#5537]
Hi,

This morning on Openshot I have no more thumbnail in the Files project area. just the names of the files.

I just open the program and add some files (videos, images, titles) in the files area.

I uninstalled openshot and cleaned the registry but the issue persist. Is it a bug ? A setting ?

![openshot-qt_Wn14DJ0DSA](https://github.com/OpenShot/openshot-qt/assets/172365710/d9f5d6d6-5404-4412-ad7c-fd3eb1233e1c)

**Comment by Colorjet3** [comment]
1.  Which version of OpenShot are you running?
2.  What operating system/version are you running?
3.  Does this happen in this project only or other projects?  If you don't have other projects, please create a test one and see if the thumbnails are showing or not.

**Comment by Blankovitch** [comment]
Hi ColorJet3

1. W10
2. 3.1.1
3. it happens for all my projects now. including the former ones.

I tried to uninstall it, deleting the settng folder... nothing works.

**Comment by Colorjet3** [comment]
Thank you @Blankovitch for the information.

Please do this:
1.  Close OpenShot if running.
2.  Make a backup of c:\users\username\.openshot_qt folder.
3.  Delete c:\users\username\.openshot_qt folder.
4.  Go to openshot.org/download/#daily and download the latest dev daily build and install it.
5.  Start OpenShot.
6.  Create a new test project.  Import a few of your source files and see if the thumbnails are working.
7.  Now open your existing projects and see if the issue has been resolved.

**Comment by Blankovitch** [comment]
Ok. I will do that and I'll let you Know

**Comment by Blankovitch** [comment]
I'm Back and the issue is the same.
It's very weird.

When i look in the .openshot_qt directory after having reinstall it (the last dayly build, or the relaase) the "Thumbnail" directory is still empty.

I suppose it's not the case normally.

**Comment by Colorjet3** [comment]
Hello @Blankovitch.

Version 3.2.1 has been released.  Please re-install following the [Clean Installation Instructions](https://github.com/OpenShot/openshot-qt/wiki/Clean-Installation-of-OpenShot).

**Comment by Blankovitch** [comment]
Thank you Colorjet, i'll try and i'll let you know.

**Comment by Colorjet3** [comment]
Hello @Blankovitch Any update on this?

**Comment by Colorjet3** [comment]
No response from OP.

**Comment by Blankovitch** [comment]
Sorry openjet. I'm on leave from work.
The issue happens on my computer work. I'll let you know if the new version
solved the problem.


Le lun. 29 juil. 2024 à 20:12, Colorjet3 ***@***.***> a
écrit :

**Comment by Blankovitch** [comment]
I come back to my office next week :)

**Comment by Blankovitch** [comment]
Hi !
Back in my office today.
The issue still happens with the 3.2.1 version. The thumbnail directory remains empty when I add some media and the previews pictures aren't appearing.

It's so weird....

**Comment by Blankovitch** [comment]
Hi !
Back in my office today.
The issue still happens with the 3.2.1 version after a clean install. The thumbnail directory remains empty when I add some medias and the preview isn't appearing.

It's so weird...

**Comment by Blankovitch** [comment]
Hi again !
Miracle !

I tried with no hope to simply reinstall Openshot from the same installer 3.2.1.

And the tumbnails are back.

It worked but I had no idea of how the problem was solved.

I realised somethingt : I was on my pro network and when it finally worked I was on a shared wifi connection from my phone for some reason.

I tried to turn back on the office network and the problem was back !

The problem came from the proxy of my pro network !!! 

Weird, strange, but solved !
